crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -768,6 +771,52 @@ impl<'w, 's> Commands<'w, 's> {
     ) -> EntityCommands {
         self.spawn(Observer::new(observer))
     }
+
+    /// Registers a system and returns a [`SystemId`] so it can later be called by [`World::run_system`].
```

Comment:
```suggestion
    /// Registers a system and returns a [`SystemId`] so it can later be called by [`Commands::run_system`].
```

Since this function is called within the context of `Commands`, should it refer to `Commands` instead of `World`?